tasks/pages/ward/[wardId].tsx

```diff
@@ -211,18 +209,24 @@ const WardOverview: NextPage = ({ language }: PropsWithLanguage<WardOverviewTran
     setContextState({ wardId })
   }, [wardId])
 
-  const createMutation = usePatientCreateMutation(organizationId, patient => {
-    const updatedContext = contextState
-    updatedContext.patient = patient
+  const createPatientMutation = usePatientCreateMutation(organizationId)
 
-    if (contextState.bedId) {
-      assignBedMutation.mutate({
-        id: contextState.bedId,
-        patientId: patient.id
+  // TOOD: can we somehow assert that the patient is not undefined here?
+  const createPatient = () => contextState.patient && createPatientMutation.mutateAsync(contextState.patient)
+    .then((patient) => {
+      // TODO: is it a good idea to have this much state in one object?, also this shouldn't be called context I think
+      setContextState({
+        ...contextState,
+        patient,
```

Comment:
@DasProffi

Reply:
we cannot assert that patient is not undefined as this indicates whether a patient will be created and if not set a patient isn't created. This might be solved by splitting the state as you suggested, but will be much effort for checking that all tests against the context stay the same.

Regarding the rename: This is the state of the provided context for children, if you prefer a different naming, I won't object

Reply:
The thing with the word context is that there exists the concept or [react contexts](https://react.dev/reference/react/createContext); I wouldn't use a name that is already taken by something else in the same domain